lnrpc/rpc.pb.go

```diff
@@ -2956,7 +2956,7 @@ func (m *QueryRoutesRequest) GetNumRoutes() int32 {
 	if m != nil {
 		return m.NumRoutes
 	}
-	return 0
+	return 10
```

Comment:
This unfortunately isn't the place to make this change as this file and others are generated automatically by the proto compiler. Instead, the check should be done at the [QueryRoutes RPC level](https://github.com/lightningnetwork/lnd/blob/832bd8101c889efb9ace39bd1f4364d34900f351/rpcserver.go#L3407).

Reply:
Thanks for the pointer @vapopov  - I've updated the PR.